src/intersections/rays.jl

```diff
@@ -153,14 +153,15 @@ function intersection(f, ray::Ray, tri::Triangle)
   det = e₁ ⋅ p
 
   # keep det > 0, modify T accordingly
-  if det > atol(det)
+  det_atol = atol(det)
```

Comment:
```suggestion
  atoldet = atol(det)
```
We don't use underscore.